integration/kubernetes/k8s-attach-handlers.bats

```diff
@@ -28,7 +31,8 @@ setup() {
 	kubectl create -f "${pod_config_dir}/test-lifecycle-events.yaml"
 
 	# Check pod creation
-	kubectl wait --for=condition=Ready pod "$pod_name"
+	cmd="kubectl wait --for=condition=Ready pod $pod_name"
```

Comment:
`kubectl wait` has a `timeout` argument with default value `30s`,  so how about to only add a `timeout` to `kubectl` command ?

```
      --timeout=30s: The length of time to wait before giving up.  Zero means check once and don't
wait, negative means wait for a week.
```